learning_journal_old/__init__.py

```diff
@@ -0,0 +1,12 @@
+from pyramid.config import Configurator
```

Comment:
You shoudl probably kill off `learning_journal_old` and all of the stuff inside it.  It isn't used anymore, right?